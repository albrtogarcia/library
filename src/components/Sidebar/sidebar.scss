.sidebar {

  &-header {
    display: flex;
    align-items: center;
    justify-content: space-between;
    gap: var(--gap-regular, 1rem);
    flex-shrink: 0;
  }
}