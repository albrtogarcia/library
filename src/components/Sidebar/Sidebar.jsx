import { Input, Icon, Button } from '../index'
import './sidebar.scss'

export const Sidebar = () => (
  <div className="sidebar">
    <div className="sidebar-header">
      <h3 className="sidebar-title">Filtros</h3>
      <Icon name="filter" />
    </div>

    <div className="sidebar-filters">
      <Input label="UbicaciÃ³n" />
    </div>
    <Button intent="primary" fill>
      Filtrar
    </Button>
  </div>
)
