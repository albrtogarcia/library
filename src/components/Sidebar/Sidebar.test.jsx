import { describe, expect, it } from 'vitest'
import { Sidebar } from './Sidebar'

describe('Sidebar', () => {
  it('matches snapshot', () => {
    const element = Sidebar()
    expect(element).toMatchSnapshot()
  })
})
