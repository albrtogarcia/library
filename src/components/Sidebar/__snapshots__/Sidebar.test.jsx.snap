// Vitest Snapshot v1, https://vitest.dev/guide/snapshot.html

exports[`Sidebar > matches snapshot 1`] = `
<div
  className="sidebar"
>
  <div
    className="sidebar-header"
  >
    <h3
      className="sidebar-title"
    >
      Filtros
    </h3>
    <Icon
      name="filter"
    />
  </div>
  <div
    className="sidebar-filters"
  >
    <Input
      label="Ubicación"
    />
    <Blueprint5.FormGroup
      label="Anunciante"
      labelFor="user-email"
    >
      <Blueprint5.InputGroup
        className="form-control"
        id="user-email"
        type="email"
      />
    </Blueprint5.FormGroup>
    <Blueprint5.FormGroup
      label="Tipo de publicidad"
      labelFor="user-email"
    >
      <Blueprint5.InputGroup
        className="form-control"
        id="user-email"
        type="email"
      />
    </Blueprint5.FormGroup>
    <Blueprint5.FormGroup
      label="Tipo de negociación"
      labelFor="user-email"
    >
      <Blueprint5.InputGroup
        className="form-control"
        id="user-email"
        type="email"
      />
    </Blueprint5.FormGroup>
  </div>
  <Blueprint5.Button
    fill={true}
    intent="primary"
  >
    Filtrar
  </Blueprint5.Button>
</div>
`;
